src/open_inwoner/scss/components/modal/_modal.scss

```diff
@@ -74,6 +74,14 @@
     @media (min-width: 768px) {
       min-width: 150px;
     }
+
+    &:focus {
+      transform: none;
+
+      &:hover {
+        transform: translateY(-1px);
+      }
```

Comment:
This has a strange effect: the button not only moves to the top but is also contracted (it looses one pixel on the right).

Reply:
Yeah this is Firefox specific (Edge/Safari/Chrome do not have this). We also have this behaviour for ALL types of buttons that are hovered (like in the registration form etc.) so we'll promote this to a separate issue, since it's unrelated to the modals, so we can finalize the modals here :-)

Reply:
In case someone wants to see the new separate issue: https://taiga.maykinmedia.nl/project/open-inwoner/issue/2293